client/src/Home/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import { Carousel } from 'antd';
+import 'antd/dist/antd.css';
+import './header.css';
+
+function Header() {
+  return (
+    <div className="header">
```

Comment:
I think Muath has the same className to another element, you should just change the class name here.

Reply:
each component in different files and each CSS just imported in the component file